cfgov/unprocessed/css/nav-secondary.less

```diff
@@ -132,176 +101,117 @@
                 .nav-secondary_item
-                  .nav-secondary_link / .nav-secondary_link.nav-secondary_link__current
+                  .nav-secondary_link /
+                  .nav-secondary_link.nav-secondary_link__current
   tags:
     - cfgov-nav-secondary
 */
 
 .nav-secondary {
+    .webfont-regular();
 
     .respond-to-max(@bp-sm-max, {
+        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
+
         background-color: @gray-10;
         border-top: 1px solid @gray-20;
         border-bottom: 1px solid @gray-20;
-        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
     });
 
-    // Inset horizontal rule between "IN THIS SECTION" text and page links.
-    & .inset-divider {
-      margin: 0 @grid_gutter-width / 2 @grid_gutter-width / 2;
-      border: 0;
-      height: 1px;
-      background-color: @gray-20;
+    &_button {
+        width: 100%;
+        padding: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+
+        .webfont-demi();
+
+        text-align: left;
+        text-transform: uppercase;
+
+        &:focus {
+            outline: 1px dotted;
+        }
 
-      // Only shown on tablet/mobile sizes.
-      .respond-to-min(@bp-lg-min, {
-        display: none;
-      });
+        .cf-icon {
+            color: @green;
+        }
+
+        .respond-to-min(@bp-med-min, {
+            display: none;
+        });
     }
 
     &_list {
         margin: 0;
         padding: 0;
-    }
-    &_item, &_item__child {
-        list-style: none;
-        .webfont-regular();
 
-        .respond-to-min(@bp-lg-min, {
-            & + & {
-                margin-top: 10px;
-            }
-        });
-    }
+        &__parents {
+            .respond-to-max(@bp-sm-max, {
+                padding-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+                margin-left: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+                margin-right: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
 
-    &_item {
-        .respond-to-min(@bp-med-min, {
-            & + &,
-            &__child {
-              margin-top: 10px;
-            }
-        });
-    }
+                border-top: 1px solid @gray-50;
+            });
+        }
 
-    &_item__parent {
-        .h4();
-        .webfont-medium();
-        margin-bottom: 0;
-        padding-bottom: 0;
-    }
+        &__children {
+            margin-left: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
 
-    &_item__child {
-        // Gutter width minus optical offset.
-        margin-left: @grid_gutter-width - 13px;
+            .respond-to-min(@bp-med-min, {
+                // Add 5px for the border to half the gutter
+                margin-left: unit( (@grid_gutter-width / 2 + 5px ) / @base-font-size-px, em) ;
+            });
+        }
     }
 
     &_link {
         .u-inline-block();
-        margin-left: unit(-15px / @font-size-large, em);
```

Comment:
If I understand the design intent correctly, we want the vertical green bar to appear out in the margin, so that the left edge of the text lines up with the left edge of the top nav & logo. This line needed to be maintained in order for that to happen.